monky.el

```diff
@@ -112,6 +113,62 @@ is usually faster if Monky runs several commands."
   :type '(choice (const :tag "Single processes" :value nil)
                  (const :tag "Use command server" :value cmdserver)))
 
+(defcustom monky-pull-args ("--config" "ui.merge=:merge")
```

Comment:
I suggest move this config option to ```monky-hg-standart-options```. In this way we can have single merge tool across all commands.